ui/main/src/app/modules/cards/components/card/card.component.spec.ts

```diff
@@ -40,6 +40,8 @@ describe('CardComponent', () => {
 
     beforeEach(async(() => {
         const routerSpy = createSpyObj('Router', ['navigate']);
+        let myrout = {... routerSpy};
```

Comment:
I discussed it with @rlg-rte, he said that was an good way to go around the test issue, he just said that maybe we could add the routerState property directly on the routerSpy object so no copy is needed.